llvm/include/llvm/TableGen/StringToOffsetTable.h

```diff
@@ -52,6 +52,33 @@ class StringToOffsetTable {
     return II->second;
   }
 
+  // Emit the string using string literal concatenation, for better readability
+  // and searchability.
+  void EmitStringLiteralDef(raw_ostream &OS, const Twine &Decl,
```

Comment:
What does `Def` mean in the name?

Reply:
Yeah, I adopted the name from `SequenceToOffsetTable`. The def means definition.